pkg/scheduler/factory.go

```diff
@@ -343,6 +343,14 @@ func MakeDefaultErrorFunc(client clientset.Interface, podLister corelisters.PodL
 			klog.Warningf("Pod %v/%v doesn't exist in informer cache: %v", pod.Namespace, pod.Name, err)
 			return
 		}
+
+		// The restful bind request of extender scheduler with webhook is not an atomic operation.
+		// If the bind request fails, the bind request to kube-apiserver in webhook may be successful, and pod info have been modified.
```

Comment:
In the case of extender, the pod may have been bound successfully, but timed out returning its response to the scheduler.
It could result in the live version to carry .spec.nodeName, and that's inconsistent with the internal-queued version.